website/docs/d/app_service_virtual_network_connection.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_virtual_network_connection"
+sidebar_current: "docs-azurerm-resource-app-service-virtual-network-connection"
+description: |-
+  Get Gateway Required Virtual Network Connection (Integrate with an App Service).
+
+---
+
+# Data Source: azurerm_app_service_virtual_network_connection
+
+Use this data source to access information about an existing App Service integrated Virtual Network Connection.
+
+## Example Usage
+
+```hcl
+data "azurerm_app_service_virtual_network_connection" "example" {
+  resource_group_name   = "example-resource-group"
+  app_service_name      = "example-appservice"
+  vnet_name             = "example-virtual-network"
+}
+
+output "app_service_virtual_network_connection_id" {
+  value = "${data.azurerm_app_service_virtual_network_connection.example.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `app_service_name` - (Required) Specifies the name of the App Service.
+
+* `resource_group_name` - (Required) The name of the resource group which the app service belongs to.

```

Comment:
COuld we use data source phrasing:
`pecifies the name of the Resource Group where the PostgreSQL Server exists.`